eng/pipelines/templates/Build_and_UnitTest.yml

```diff
@@ -226,7 +225,6 @@ steps:
     configuration: "$(BuildConfiguration)"
     msbuildArguments: "/t:AfterBuild"
     maximumCpuCount: true
-  condition: and(succeeded(), or(eq(variables['IsOfficialBuild'], 'true'), eq(variables['BuildRTM'], 'true')))  #skip this task for nonRTM in private build
```

Comment:
I think we still need succeeded check.

Reply:
When you don't specify a condition, the default is `succeeded()`.

Reply:
I think the default condition has succeeded check, according to `Only when all previous dependencies have succeeded. This is the default if there is not a condition set in the YAML `in [this doc](https://docs.microsoft.com/en-us/azure/devops/pipelines/process/conditions?view=azure-devops&tabs=yaml).